### DISCUSSION TITLE
Qwen-Image facial similarity for the same prompts with different seeds; similarity goes away after fine-tuning

### DISCUSSION THREAD

**Discussion by BelieveDiffusion** [kohya-ss/musubi-tuner#804] [Q&A]
Hi there,

I'm experimenting with musubi-tuner for fine-tuning Qwen-Image on a dataset of 18k hand-annotated high-quality images, with the goal of improving the "natural"-ness of the images (skin texture etc). I'm seeing a curious behavior in Qwen-Image, which I haven't seen in other models, and I wanted to see if it's expected.

What I'm seeing is that Qwen-Image will _always_ generate a person with the same face and hair identity for the same prompt with different seeds. So if you generate a batch of eight images from a single prompt of a person, then that person will look exactly the same (apart from expression) in all eight images.

What I'm _also_ seeing is that if I fine-tune the model on my own dataset, then (at least after two epochs), sometimes the images generated by my fine-tune will match that same original person exactly, and sometimes they will not match them at all. (I think it tends towards "not matching" the more I include phrases from my own tagging taxonomy, but I haven't proven that conclusively.)

The really confusing thing is that when the person doesn't match, it's never a slight evolution on from the original Qwen-Image person; it's always someone who looks completely different.

Is this expected? Is it a good thing? Is it a sign that my training is working as expected (moving away from the uniformity), or not working as expected (Qwen loses its ability to generate the same person for the same prompt with different seeds)?

Examples below, all generated in ComfyUI with the following settings:

- Model: qwen_image_bf16.safetensors (original) and my own two-epoch fine-tune over 18k images (custom)
- Sampler: heun
- Scheduler: beta
- Seed: 1234
- Steps: 20
- CFG: 4.0
- Size: 1056x1584

In each case, I generated a batch of four images (Qwen-Image on the left, my fine-tune on the right).

## Prompt 1

```a woman sitting in a coffee shop in san francisco, brown hair, smiling, looking at camera, drinking a cup of coffee, wearing a gray knitted sweater and a long black skirt```

All four fine-tune images match the original character from Qwen-Image.

![CoffeeShop_Prompt01](https://github.com/user-attachments/assets/5df77ea5-783e-4e30-91c4-05e89eda70e9)

## Prompt 2

```a woman sitting in a coffee shop in san francisco, pixie-cut straight brown hair, smiling, looking at camera, wearing a gray knitted sweater, hands on thighs, dangly earrings, bracelet```

All four fine-tune images are very different to the original character.

![CoffeeShop_Prompt02](https://github.com/user-attachments/assets/0c0b4dbf-428a-4155-b578-6abd64ffa67d)

## Prompt 3

```a woman sitting in a coffee shop in san francisco, straight brown hair, smiling, looking at camera, wearing a gray knitted sweater, hands on thighs```

All four fine-tune images are very different to the original character.

![CoffeeShop_Prompt03](https://github.com/user-attachments/assets/fa897a38-cf35-496a-bc99-7c99255ab00c)

## Prompt 4

```a woman sitting in a coffee shop in san francisco, straight brown hair, smiling, looking at camera, wearing a gray knitted sweater```

All four fine-tune images are very different to the original character.

![CoffeeShop_Prompt04](https://github.com/user-attachments/assets/6b6db75b-bb6a-42de-b2fe-5f4829e2f607)

## Prompt 5

```a woman sitting in a coffee shop in san francisco, straight brown hair, smiling, wearing a gray knitted sweater```

Three images match the original character; one is very different.

![CoffeeShop_Prompt05](https://github.com/user-attachments/assets/b78bb8a1-24bd-49ad-be63-c62eafdb9022)

## Prompt 6

```a woman sitting in a coffee shop in san francisco, straight brown hair, wearing a gray knitted sweater```

Two images match the original character; two are very different.

![CoffeeShop_Prompt06](https://github.com/user-attachments/assets/35b190ef-59b7-40b6-a561-93e4e9c7ef30)

--

I'm particularly curious to know how I should rate the "success" of my fine-tuning, given this seemingly binary switch for some prompts and images but not others.

**Comment by BelieveDiffusion** [comment]
Fun fact: I have _not_ seen this "sudden change" behavior when fine-tuning the newer [Qwen-Image-2512](https://huggingface.co/Qwen/Qwen-Image-2512) model.

**Comment by JigenD** [comment]
I have observed this and definitely agree this occurs. My speculation is qwen-image has a little DPO applied which caused this behavior and by training upon it, the DPO is effectively unlearned / ablated. And I assume that Qwen-Image-2512 has much more of this DPO applied. I am saying DPO specifically because I believe they mentioned that in the official paper, but correct me if I'm wrong.

**Comment by BelieveDiffusion** [comment]
Oh, that's really interesting! Yes, that would make sense. I also noticed a big difference in detail / grain / skin tone in my fine-tune, which might be another part of DPO that was lost.

Naive question: are there any techniques for training / fine-tuning that preserve the DPO?

**Comment by BelieveDiffusion** [comment]
Yep, Quen-Image does indeed use DPO (as stated in [section 4.2.2 of their white paper](https://qianwen-res.oss-cn-beijing.aliyuncs.com/Qwen-Image/Qwen_Image.pdf)).
